frontend/src/app/modules/bim/ifc_models/bcf/list-container/bcf-list-container.component.ts

```diff
@@ -103,10 +100,19 @@ export class BcfListContainerComponent extends WorkPackageListViewComponent impl
       return;
     }
 
-    // Otherwise, always open all links in the details view
-    this.$state.go(
-      splitViewRoute(this.$state),
-      { workPackageId: event.workPackageId, focus: true }
-    );
+    this.goToWpDetailState(event.workPackageId, this.$state.params.cards, true);
+  }
+
+  goToWpDetailState(workPackageId:string, cards:boolean, focus?:boolean) {
+    // Show the split view when there is a viewer (browser)
+    // Show only wp details when there is no viewer, plugin environment (ie: Revit)
+    const stateToGo = this.viewer.shouldShowViewer ?
+                        splitViewRoute(this.$state) :
+                        'bim.partitioned.show';
+    // Passing the card param to the new state because the router doesn't keep
+    // it when going to 'bim.partitioned.show'
+    const params = { workPackageId, cards, ...focus && {focus} };
```

Comment:
What does this do: `...focus && {focus}` ? either of the two would suffice I'd reckon

Reply:
Thanks for your review @oliverguenther . It only passes the 'focus' param when it is truthy. I didn't know what the param was for so I tried to not break anything on the wp-list-component.ts by passing it as a falsy value. 
Changed.